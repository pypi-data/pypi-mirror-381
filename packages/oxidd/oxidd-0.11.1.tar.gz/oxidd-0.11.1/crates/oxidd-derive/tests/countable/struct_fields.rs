use oxidd_derive::Countable;

#[derive(Countable)]
struct Tag1(u8);

#[derive(Countable)]
struct Tag2 {
    field: u8,
}

fn main() {}
