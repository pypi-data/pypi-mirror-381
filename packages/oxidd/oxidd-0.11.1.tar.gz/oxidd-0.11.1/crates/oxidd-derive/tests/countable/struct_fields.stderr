error: `Countable` can only be derived for `struct`s with zero fields
 --> tests/countable/struct_fields.rs:4:12
  |
4 | struct Tag1(u8);
  |            ^^^^

error: `Countable` can only be derived for `struct`s with zero fields
 --> tests/countable/struct_fields.rs:7:13
  |
7 |   struct Tag2 {
  |  _____________^
8 | |     field: u8,
9 | | }
  | |_^
